packages/kubernetes-1.21/kubelet-config

```diff
@@ -116,6 +116,10 @@ serializeImagePulls: false
 serverTLSBootstrap: {{settings.kubernetes.server-tls-bootstrap}}
```

Comment:
We may want to only do `serverTLSBootstrap` if we don't have the cert + private key, even if it's set to "true", because otherwise the bootstrap process will either do nothing (?) or else bootstrap a `/var/lib/kubelet/pki/kubelet-server-current.pem` certificate that (confusingly) isn't used.